bazel/ray_deps_setup.bzl

```diff
@@ -236,13 +236,13 @@ def ray_deps_setup():
     auto_http_archive(
         name = "com_github_grpc_grpc",
         # NOTE: If you update this, also update @boringssl's hash.
-        url = "https://github.com/grpc/grpc/archive/refs/tags/v1.42.0.tar.gz",
-        sha256 = "b2f2620c762427bfeeef96a68c1924319f384e877bc0e084487601e4cc6e434c",
+        url = "https://github.com/grpc/grpc/archive/refs/tags/v1.43.0.tar.gz",
```

Comment:
Can we try v1.43.2 which has some bug fixes?

Reply:
O.o just saw it got released. Yeah might has well update to 1.43.2.

Reply:
Done.